kratos/python_scripts/process_factory.py

```diff
@@ -18,31 +19,52 @@ def ConstructListOfProcesses( self, process_list ):
                 raise NameError('"python_module" must be defined in your parameters. Check all your processes')
 
             # python-script that contains the process
-            python_module_name = item["python_module"].GetString()
+            python_module_name = self._GetFullModuleNameName(item)
+            
+            python_module = import_module(python_module_name)
+            p = python_module.Factory(item, self.Model)
+            constructed_processes.append( p )
 
-            if item.Has("kratos_module"): # for Kratos-processes
-                """Location of the process in Kratos; e.g.:
-                - KratosMultiphysics
-                - KratosMultiphysics.FluidDynamicsApplication
-                """
+        return constructed_processes
+
+    def GetMapOfRequiredHistoricalVariables( self, process_list):
+        required_variables_map = defaultdict(list)
+        for i in range(0,process_list.size()):
+            item = process_list[i]
```

Comment:
```suggestion
        for item in process_list:
```
Is there a specific reason not to have the more pythonic way?

Reply:
copy-paste :D

Reply:
doesn't work => #1466

Reply:
hmm tricky. Reverted.

Reply:
Oh.. thanks :)